app/actions/pure.js

```diff
@@ -158,5 +158,15 @@ export default {
     ADMIN_UPDATE_DOCUMENT_FAILURE: 'ADMIN_UPDATE_DOCUMENT_FAILURE',
     ADMIN_DELETE_DOCUMENT_REQUEST: 'ADMIN_DELETE_DOCUMENT_REQUEST',
     ADMIN_DELETE_DOCUMENT_SUCCESS: 'ADMIN_DELETE_DOCUMENT_SUCCESS',
-    ADMIN_DELETE_DOCUMENT_FAILURE: 'ADMIN_DELETE_DOCUMENT_FAILURE'
+    ADMIN_DELETE_DOCUMENT_FAILURE: 'ADMIN_DELETE_DOCUMENT_FAILURE',
+
```

Comment:
Is there a reason for this space?

Reply:
I just saw that other things like mentorship had their own space block so it seemed natural; if you think that shouldn't be there that works